tuist/tuist
# Align ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES with the Xcode default

Resolves <https://github.com/tuist/tuist/issues/6989>

### Short description 📝
I modified the tuist/XcodeProj file to resolve this issue.

### How to test the changes locally 🧐
1. clone XcodeProj project to use local
2. modified local XcodeProj, see this [pr](https://github.com/tuist/XcodeProj/pull/881)
3. modified Package.swift to use local XcodeProj
4. tested in fixtures/workspace_with_test_locale using `tusit generate`

And I also changed Target init in fixtures/workspace_with_test_locale/Project.swift to tusit4.
Please let me know if there's anything wrong.


### Contributor checklist ✅


### Reviewer checklist ✅
